src/app.pcss

```diff
@@ -36,7 +36,8 @@ body {
     --border: 214.3 31.8% 91.4%;
     --input: 214.3 31.8% 91.4%;
 
-    --primary: 250 93.7% 68.6%;
+    /* --primary: 250 93.7% 68.6%; */
+    --primary: #7d64ff;
```

Comment:
This didn't work like I assume it would. Do I need to provide HSL values?

Reply:
I'm not knowledgeable in this part but worth trying with the same value system as before